config.py

```diff
@@ -96,8 +96,9 @@ def check_set_node_conf(name, default, node_conf):
     payment_methods.append(method_config)
 
 supported_currencies = get_opt("supported_currencies", ["USD"])
-if "BTC" in supported_currencies:
-    supported_currencies.append("sats")
+if "BTC" and not "sats" in supported_currencies:
+    BTC_index = supported_currencies.index("BTC")
+    supported_currencies.insert(BTC_index, "sats")
```

Comment:
Why this change? I personally prefer alphabetical order. Besides, seems to be out of scope of this styling change PR.

Reply:
Right sorry, didn't mean to have that commit on this pull request. I have removed it.  With that said I do not believe that the supported_currency list orders them alphabetically i believe it is just ordered how they are in the config.toml file and then sats is appended to the very end of that list.

Reply:
Ahh, right, I always add them to `config.toml` in alphabetal order. And then "sats" actually go after USD, which is not alphabetical.

Reply:
I agree it is worth having some consistency in the order across instances of Satsale, and I do agree that alphabetical order is clear and consistent but I think there is some value in trying to keep the bitcoin options together, having sats as the default selection initially but then being both last on the list and so far away from BTC seems a bit confusing as a list of fiat options gets large